src/collective/polls/tests/test_poll.robot

```diff
@@ -61,6 +64,38 @@ Test portlet poll
     Page Should Contain Element  css=#portal-column-two dl.votePortlet h3
     Page Should Not Contain Element  css=#portal-column-two dl.votePortlet h3 a[href$="test-poll"]
 
+Test tile poll
+    [Tags]  Expected Failure
+
+    Enable Autologin as  Site Administrator
+    Go to Homepage
+
+    Add Poll in Public Folder  Test Folder  Test Poll
+    Go to Homepage
+
+    Create Cover  Title  Description  Empty layout
+
+    # add a poll tile to the layout
+    Edit Cover Layout
+    Add Tile  ${polls_tile_location}
+    Save Cover Layout
+
+    # as tile is empty, we see default message
+    Compose Cover
+    Page Should Contain  Poll Tile
+
+    # drag&drop a Poll 
+    Open Content Chooser
+    Click Element  link=Content tree
+    Click Element  css=.ui-draggable a[href*=test-folder]
+    Drag And Drop  css=${poll_selector}  css=${tile_selector}
+    Wait Until Page Contains Element  css=div.poll-tile.vote-container
+
+    # move to the default view and check tile persisted
+    Click Link  link=View
+    Page Should Contain Image  css=div.poll-tile.vote-container
```

Comment:
This test is breaking here (line 96)
It looks like the tile is not added into page.. need help to figure out what is wrong here.

After this part the test should make a vote and ensure that the results are displayed.